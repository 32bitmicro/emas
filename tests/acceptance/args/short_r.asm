lwt r0, 0
lwt r1, 1
lwt r2, -1
lwt r3, 63
lwt r4, -63
