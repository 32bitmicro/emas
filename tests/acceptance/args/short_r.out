@ 0x0000 : 0xd400  /  110 101 0 000 000 000  /  54272
@ 0x0001 : 0xd441  /  110 101 0 001 000 001  /  54337
@ 0x0002 : 0xd681  /  110 101 1 010 000 001  /  54913
@ 0x0003 : 0xd4ff  /  110 101 0 011 111 111  /  54527
@ 0x0004 : 0xd73f  /  110 101 1 100 111 111  /  55103
